### ISSUE TITLE
PUSH label returns an error

### ISSUE THREAD

**Issue by atar-axis** [TUM-LRR/Jasmin#10]
The following code returns "Operand must be at 2 two bytes large" on line 4.
It's a bug, isn't it?

```
miau:
DW 0x1234

PUSH miau
```

**Comment by meyerm** [comment]
The first example `dw 0x1234` as well as the second one `dw 1234` should both work - both values can be extended to a full data word. The third, on the other hand, is wrong, as `dw` only declares a 2-byte date while `dword` refers to a 4-byte date.

Trying them out in Jasmin **1.5.8** as well as the current **HEAD** shows that Jasmin handles all cases correctly. Did I misunderstand something?

**Comment by atar-axis** [comment]
Sorry, I forgot the most important thing and confused DD with DW yesterday.
This is the example i wanted to show you:

`miau:
DD 0x1234

PUSH miau`

miau in `PUSH miau` should be replaced by a 32-bit address, right?
but jasmin complains that the operand is too short!

If I insert a DWORD before miau, like `PUSH DWORD miau`, it works.
Shouldn't it also work without the DWORD thing?

**Comment by meyerm** [comment]
You are right. In the "real world" a `push LABEL` puts that address onto the stack and in Jasmin this should be the case, too.

Even though we probably should also think about our Harvard approach (as already mentioned in #5) in case somebody want to push a code line onto the stack... But first things first.
